lib/trie/child_storage.go

```diff
@@ -41,27 +43,27 @@ func (t *Trie) GetChild(keyToChild []byte) (*Trie, error) {
 }
 
 // PutIntoChild puts a key-value pair into the child trie located in the main trie at key :child_storage:[keyToChild]
-func (t *Trie) PutIntoChild(keyToChild, key, value []byte) error {
+func (t *Trie) PutIntoChild(keyToChild, key, value []byte) (*common.Hash, error) {
```

Comment:
Why are returning the root hash of the child trie? It doesn't seem to be used anywhere right?